tests/test_reverse_m2m_relation_crudl_operations_are_supported.py

```
@@ -0,0 +1,242 @@
+        name="Some publisher",
+        address="Some address",
+    )
+    book: models.Book = models.Book.objects.create(
+        title="Some book",
+        isbn="9783161484100",
+        publication_date="2021-01-01",
+        publisher=publisher,
+    )
+    author: models.Author = models.Author.objects.create(
+        name="Some author",
+        # birth_date="1990-01-01",
```

Ruff (0.8.2)

Found commented-out code

Remove commented-out code

(ERA001)